src/Controller/PagesController.php

```diff
@@ -66,5 +69,7 @@ public function display(...$path)
             }
             throw new NotFoundException();
         }
+
+        return null;
```

Comment:
Should we `return $this->render()` instead?

Reply:
Yeah, if we are going ot add return typehint it's better to explicitly render and return response, else don't add the typehint.

Reply:
OK, I adjusted it.